editor/plugins/game_view_plugin.cpp

```diff
@@ -287,10 +429,44 @@ void GameView::_notification(int p_what) {
 			select_mode_button[RuntimeNodeSelect::SELECT_MODE_LIST]->set_button_icon(get_editor_theme_icon(SNAME("ListSelect")));
 
 			hide_selection->set_button_icon(get_editor_theme_icon(hide_selection->is_pressed() ? SNAME("GuiVisibilityHidden") : SNAME("GuiVisibilityVisible")));
+			keep_aspect_button->set_button_icon(get_editor_theme_icon(SNAME("KeepAspect")));
+			embed_options_menu->set_button_icon(get_editor_theme_icon(SNAME("GuiTabMenuHl")));
 
 			camera_override_button->set_button_icon(get_editor_theme_icon(SNAME("Camera")));
 			camera_override_menu->set_button_icon(get_editor_theme_icon(SNAME("GuiTabMenuHl")));
 		} break;
+
+		case NOTIFICATION_READY: {
+			if (DisplayServer::get_singleton()->has_feature(DisplayServer::FEATURE_WINDOW_EMBEDDING)) {
+				// Embedding available.
+				embed_on_play = EditorSettings::get_singleton()->get_project_metadata("game_view", "embed_on_play", true);
+				make_floating_on_play = EditorSettings::get_singleton()->get_project_metadata("game_view", "make_floating_on_play", true);
+				keep_aspect_button->set_pressed(EditorSettings::get_singleton()->get_project_metadata("game_view", "keep_aspect", true));
```

Comment:
The default value shouldn't make the button be in the toggled state, as it makes it unnecessarily stand out.

Reply:
I'm not sure I understand what you suggest exactly? That the keep aspect if not enabled by default or to change it for a "Do not keep aspect"? I'm not sure what is the issue, there are other toolbar button in the other editors that have the same behavior.

Reply:
I meant the latter, but thinking again, it's not that important.